src/Elcodi/Bundle/CartBundle/Resources/config/eventListeners.yml

```diff
@@ -11,7 +11,7 @@ services:
             - @elcodi.cart.manager
             - @elcodi.wrapper.currency
             - @elcodi.converter.currency
-            - @elcodi.manager.configuration
+            - "@=elcodi_config('product.use_stock', false)"
```

Comment:
What's with the `false`? `elcodi_config` has no second parameter in expression language.